debian/postinst

```diff
@@ -24,15 +24,15 @@ upgrading=0
 install_package() {
   PIP=/opt/stackstorm/st2/bin/pip
   WHEELSDIR=/opt/stackstorm/share/wheels
-  ${PIP} install --find-links ${WHEELSDIR} --no-index --quiet --upgrade st2-enterprise-rbac-backend
+  ${PIP} install --find-links ${WHEELSDIR} --no-index --quiet --upgrade st2-rbac-backend
 }
 
 enable_rbac() {
   crudini --set ${ST2_CONFPATH} rbac enable 'True'
 }
 
 configure_rbac() {
-  crudini --set ${ST2_CONFPATH} rbac backend 'enterprise'
+  crudini --set ${ST2_CONFPATH} rbac backend 'default'
```

Comment:
Later on in the file it states that we still set rbac.backend to enterprise, to correctly handle upgrades from pre v3.3.0 (line 58). But we now set it to default - so I'm not sure if this is right, or comment below needs more explanation?

Reply:
Thanks. I'll update the comments below in this file. This value here tells st2 which RBAC backend to use. Previously, this backend is named "enterprise". In this patch, I renamed the backend to "default".